androidbrowserhelper/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 2.6.2
+
+* [#520](https://github.com/GoogleChrome/android-browser-helper/pull/520) Skip animation as
```

Comment:
In general, it would be good to have the first sentence be a summary of the change. "Skip animation as default" is ambiguous; does it dismiss the animation and introduce glitches? What animation is skipped? What is the benefit and what does the user need to worry about for the upgrade?

In my last comment I suggested "The TWA launcher will no longer wait for the animation to finish before launching the Custom Tabs Activity, which improves launch time". I still think this better conveys the nuance that this is a performance optimization and more concise. And perhaps we can additionally state "this change does not affect the user-visible launch animation".